<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDressShop</class>
 <widget class="QWidget" name="UserDressShop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserDressShop</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 127);
background-color: rgb(255, 255, 127);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="repoBasket">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 85, 255, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(170, 235, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BuyDress">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>BUY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="shoppingBasket">
       <property name="styleSheet">
        <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="shopping_basket_button">
     <property name="styleSheet">
      <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(170, 255, 173, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>See Shopping Basket</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
